app/models/errors.rb

```diff
@@ -1,6 +1,5 @@
 module Errors
   class InsufficientAllotment < StandardError
-    # TODO: This should be removed once other models no longer depend on it; it should be encapsulated and accessed through an interface
```

Comment:
I couldn't understand this TODO enough to take action on it - which means it's complex enough we should probably move it into a Github Issue?